spring-boot/src/test/java/org/springframework/boot/context/config/ConfigFileApplicationListenerTests.java

```diff
@@ -606,7 +606,8 @@ public void absoluteResourceDefaultsToFile() throws Exception {
 		String location = new File("src/test/resources/specificlocation.properties")
 				.getAbsolutePath();
 		TestPropertySourceUtils.addInlinedPropertiesToEnvironment(this.environment,
-				"spring.config.location=" + location);
+				// java.util.Properties needs the values escaped
+				"spring.config.location=" + location.replaceAll("\\\\", "\\\\\\\\"));
```

Comment:
I've attempted to fix that one in a different way. I'm still hitting other Windows issues at the moment but I hope to get our Windows CI green again tonight.